### PR TITLE
Wp 473

### PR DESCRIPTION
**Pull request by kwlodarska**
Mobile modification of test bed application. I created media query for styles and additional javascript file only for mobile look. Unfortunately I had to modify a little orginal html code (i.e. remove tables) but then I style it to look like before for PC version.

Jira task:
http://jira.webinos.org/browse/WP-473

### CODE REVIEW

**Comment by ghost** [comment]
Hi Katarzyna,

Please update the git commit messages according to the instructions on the dev portal (more meaningful commit messages, reference jira issue).
Thanks!

File: webinos/test/client/client.html [hunk 1]
```diff
@@ -62,7 +63,10 @@
         if (pzpId && typeof pzpId.split('/')[1] === "undefined") {
           $('#enableLogin').html("<a href='#login' id='login'> Connect to Your PZH </a>");
           $("#login").click(function(){
-           setPage("Login to your PZH", "login.html");
+            setPage("Login to your PZH", "login.html");
```
Comment (habibvirji) [inline webinos/test/client/client.html:66]: indentation is off

File: webinos/test/client/login.html [hunk 2]
```diff
@@ -25,8 +27,9 @@
     });
 
     function displayLogin(html_input) {
-      $('#main').html('<article class="module width_full"><header><h3> Login to your personal zone hub </h3></header><div class="module_content">'+ html_input.payload.message +
+      $('#main').html('<article class="module width_full"><header><h3> Login to Your PZH </h3></header><div class="module_content">'+ html_input.payload.message +
```
Comment (habibvirji) [inline webinos/test/client/login.html:30]: your personal zone hub looks correct

**Comment by habibvirji** [comment]
@kwlodarska can you please resubmit as the newer pull request requires updates to your branch

**Pull request closed without merging**